pkg/Elmah.AspNet.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+        <id>elmah.aspnet</id>
+        <version>2.0.0-alpha001</version>
+        <title>ELMAH AspNet Library</title>
```

Comment:
This should read simply “ELMAH for ASP.NET”

Reply:
Changed